packages/headless/src/controllers/context/headless-context.ts

```diff
@@ -1,7 +1,7 @@
 import {Engine} from '../../app/headless-engine';
 import {buildController, Controller} from '../controller/headless-controller';
 import {
-  Context as ContextPayload,
+  ContextPayload,
```

Comment:
I renamed the internal interface. ApiExtractor ignores aliases, and this made it conflict with the `Context` controller interface, resulting in an infinite loop. The `ContextPayload` interface was not exported, so this rename should not be breaking.